cardinal-components-entity/src/main/java/nerdhub/cardinal/components/api/util/RespawnCopyStrategy.java

```diff
@@ -83,7 +84,22 @@
      */
     RespawnCopyStrategy<Component> NEVER_COPY = (from, to, lossless, keepInventory) -> {};
 
+    /**
+     * Copies data from one component to the other.
+     *
+     * <p> If {@code to} implements {@link CopyableComponent}, its {@link CopyableComponent#copyFrom(Component)}
+     * method will be called, otherwise data will be copied using NBT serialization.
+     *
+     * @param from the component to copy data from
+     * @param to the component to copy data to
+     * @param <C> the common component type
+     */
+    @SuppressWarnings({"unchecked", "rawtypes"})
```

Comment:
Why would it complain about unchecked casts here?

Reply:
Because `CopyableComponent` is a raw type.